### PR TITLE
Fix token renew and update UIs

### PR DESCRIPTION
**Pull request by charithag**
- Fix issue in token renew
- Update breadcrumbs and navbaractions

### CODE REVIEW

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js [hunk 1]
```diff
@@ -20,16 +20,30 @@
  * This backendServiceInvoker contains the wrappers for back end jaggary calls.
  */
 var backendServiceInvoker = function () {
+    var log = new Log("/app/modules/backend-service-invoker.js")
     var publicXMLHTTPInvokers = {};
     var publicWSInvokers = {};
     var publicHTTPClientInvokers = {};
     var IS_OAUTH_ENABLED = true;
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:27]: Why can't this be treat as a normal var like isOAuthEnabled?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js [hunk 2]
```diff
@@ -128,7 +130,7 @@ var backendServiceInvoker = function () {
         header.setValue(constants.APPLICATION_JSON);
         httpMethodObject.addRequestHeader(header);
         if (IS_OAUTH_ENABLED) {
-            var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
+            var accessToken = getAccessToken();
             if (!(!accessToken.trim())) {
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:134]: What is the logic here? Having if (accessToken.trim()) is enough here as the double negation operators cancels each other.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js [hunk 3]
```diff
@@ -169,7 +171,7 @@ var backendServiceInvoker = function () {
         var wsRequest = new ws.WSRequest();
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
-            var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
+            var accessToken = getAccessToken();
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:174]: What is the logic here? Having if (accessToken) is enough here as the double negation operators cancels each other.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js (hunk 1 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:27]: Moved to a constant in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js (hunk 2 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:134]: Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js (hunk 3 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js:174]: Ack, Moved to a constant in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1

**Pull request merged**